### DISCUSSION TITLE
filecoin 多签 multisig

### DISCUSSION THREAD

**Discussion by jc0803kevin** [filecoin-project/community-china#107] [Bug讨论]
哪位大佬知道filecoin 多签 multisig，请教一下。

**Comment by rleor** [comment]
知道啊

**Reply by jc0803kevin** [reply]
@rleor 大佬用什么语言实现的啊

**Reply by jc0803kevin** [reply]
@rleor 目前已经用Java实现了，

**Reply by maxvint** [reply]
@jc0803kevin Filecoin lotus本身就通过多签功能哦，看一下官方文档：https://docs.filecoin.io/get-started/lotus/multisig/#inspect-a-multisig-wallet

**Comment by ohash-ai** [comment]
目前已有钱包集成了多签功能，可以去下载体验下。https://filwallet.io (需要梯子)

**Reply by jc0803kevin** [reply]
@pengxiankaikai 目前还没有开源

**Comment by GZpowerleader** [comment]
有没有需要硬件的 VX18589289061
